src/views/v2/index.php

```diff
@@ -1,8 +1,9 @@
 <?php
 use Tribe\Events\Views\V2\Template_Bootstrap;
-
-get_header();
-
-echo tribe( Template_Bootstrap::class )->get_view_html();
-
-get_footer();
\ No newline at end of file
+// @TODO: We're setting the div wrapper over here, but we'll need to revise this.
+get_header();?>
+<div class="tribe-common tribe-events">
```

Comment:
These need to be included inside of each view, not here.

Reply:
Is there any specific reason for this? This wrapper must be part of all views, because it's the one that "give sense" to all of our styles. Wouldn't it make sense not to repeat ourselves on each view and have it outside?